.github/workflows/R-CMD-check.yaml

```diff
@@ -171,6 +172,9 @@ jobs:
         shell: Rscript {0}
         run: |
           pak::local_system_requirements(execute = TRUE)
+          pak::pkg_system_requirements("renv", execute = TRUE)
+          pak::pkg_system_requirements("rsconnect", execute = TRUE)
```

Comment:
Just curious, it this what gives us the sys libs for deployment?

Reply:
yup. need to make sure curl and friends are installed. there's a lot of overlap and it _may not_ be absolutely necessary, but it's safe to do